schema/test/unit/computed_columns/project_next_milestone_due_date_test.sql

```diff
@@ -1,21 +1,17 @@
 begin;
-
 select plan(3);
 
 /**
-mock data
-create mock project
-create mock project revision
-create mock report type
-create mock reporting requirement (several)
 test cases:
 start with submitted reporting requirement (no upcoming), test that that's what we get back
 add an upcoming reporting requirement, test that that's what we get back (coalesce should give back this one if it exists)
 add several upcoming reports, make sure we get the next one
 -> use create project in create project revision functions rather than inserts
 
  **/
-
+truncate table cif.cif_user cascade;
```

Comment:
Why is this new setup necessary?

Reply:
because I wasn't able to run this test alone and had to run `make db_unit_test` everytime for a change.